sdks/rust/src/protocol/payload.rs

```diff
@@ -2680,4 +2839,334 @@ mod tests {
         let payload = SabrePayload::from_bytes(&bytes).unwrap();
         assert_eq!(payload, original);
     }
+
+    #[test]
+    // check that a create contract can be converted -> sabre payload builder -> transaction
+    // builder -> transaction
+    fn create_contract_into_transaction() {
+        let signer = HashSigner::default();
+
+        let txn_pair = CreateContractActionBuilder::new()
+            .with_name("TestContract".to_string())
+            .with_version("0.1".to_string())
+            .with_inputs(vec!["test".to_string(), "input".to_string()])
+            .with_outputs(vec!["test".to_string(), "output".to_string()])
+            .with_contract(b"test".to_vec())
+            .into_payload_builder()
+            .expect("failed to convert to payload builder")
+            .into_transaction_builder(&signer)
+            .expect("failed to convert to convert to transaction builder")
```

Comment:
failed to convert to convert to transaction builder -> failed to convert to transaction builder (and below)

Reply:
Fixed